### ISSUE TITLE
vignettes vs. chapters

### ISSUE THREAD

**Issue by nhejazi** [tlverse/acic2019-workshop#3]
## Vignettes
* Text style: bullet points
* instructor-guided
* etherpad (Software Carpentry)

## Book Chapters
* Identifiability conditions for a given parameter
* Text style: explanatory prose / long-form
* self-guided
* End-of-chapter exercises

## Both
* different levels of exercises
* learning objectives at beginning

## Open Questions
* Simulations?

**Comment by rachaelvphillips** [comment]
I still think this issue is relevant. 

**tl;dr** I think there needs to one website dedicated to the handbook and another website dedicated to the workshop materials. 

**Detail**
- At its current state, the handbook is odd and confusing with the workshop outline and "about this workshop" section. I think the handbook should be a seperate entity, tailored to an online audience. 
- I do not think we should present the handbook chapters while we're teaching in Montreal (too much text, not learner centered). The workshop materials should be tailored to an in person, actively engaged audience.
- Still, as Jade mentioned on Monday, I think the workshop attendees would appreciate access to the handbook during the workshop for reference/background. So, I think the handbook could be a nice companion to the workshop materials, and on the front page of the workshop materials we can provide a link to access the handbook?
- Of course, this is more work since it requires two different versions of the same concept. However, I think it will be worth it in the long run. I can organize/differentiate the outlines for the two webpages, if that would be helpful to you @nhejazi.

_What do you all think?_ 
If I am the only one that feels this way, then we should disregard this idea.

Happy to discuss more on Monday : )
